Proposal for the basic structure of a simulation study.md

```diff
@@ -0,0 +1,49 @@
+# Proposal for the basic structure of a simulation study
+
+### Final Structure
+
+1. *Defining aims & objectives (including hypotheses)*
+2. *Simulation procedure /Design & Data-generation mechanism*
```

Comment:
I suggest to define this step in more detail or disentangle its elements further:
- the exact experimental design, i.e which factor combinations are modulated.
- population model specification (in accordance with step 8)
- choosing values of population
These steps represent distinct decision-making processes and it could be beneficial to separately showcase how to delegate them to collaboration.

Reply:
given the here suggested structure I suggest to also incorporate step 3 this step as a distinctive but related sub step to this.

Reply:
The details you mentioned are contained in later steps. Resolving #7 and #8 should solve this